test/suite/test_oligarch07.py

```diff
@@ -0,0 +1,143 @@
+        os.mkdir('follower/foo/follower')
+        os.mkdir('follower/bar/follower')
+
+        #
+        # Part 1: Create an oligarch table and insert some data to the leader.
+        #
+        self.pr("create oligarch tree")
+        self.session.create(self.uri, self.create_session_config)
+
+        self.pr("create second WT")
+        # TODO figure out self.extensionsConfig()
+        conn_follow = self.wiredtiger_open('follower', self.extensionsConfig() + ',create,' + self.conn_base_config + 'oligarch=(role="follower")')
+        session_follow = conn_follow.open_session('')
+        session_follow.create(self.uri, self.create_session_config)
+
+        self.pr('opening cursor')
+        cursor = self.session.open_cursor(self.uri, None, None)
+
+        for i in range(self.nitems):
+            cursor["Hello " + str(i)] = "World"
+            cursor["Hi " + str(i)] = "There"
+            cursor["OK " + str(i)] = "Go"
+            if i % 100 == 0:
+                time.sleep(1)
+            if i == 0:
+                cursor_follow1 = session_follow.open_cursor(self.uri, None, None) # TODO needed so we make the metadata watcher thread earlier
+
+        # Ensure that all data makes it to the follower.
+        cursor_follow1.close()
+        cursor.close()
+        time.sleep(1)
+        self.session.checkpoint()
+        session_follow.checkpoint()
+        time.sleep(2)
+
+        #
+        # Part 2: The big switcheroo
+        #
+        self.conn.reconfigure("oligarch=(role=\"follower\")")
+        conn_follow.reconfigure("oligarch=(role=\"leader\")")
+        time.sleep(2)
+
+        #
+        # Part 3: insert content to old follower
+        #
+        cursor = session_follow.open_cursor(self.uri, None, None)
+        for i in range(self.nitems):
+            cursor["* Hello " + str(i)] = "World"
+            cursor["* Hi " + str(i)] = "There"
+            cursor["* OK " + str(i)] = "Go"
+            if i % 100 == 0:
+                time.sleep(1)
+            if i == 0:
+                cursor_follow1 = self.session.open_cursor(self.uri, None, None)
+
+        cursor.close()
+        cursor_follow1.close()
+        time.sleep(1)
+        session_follow.checkpoint()
+        self.session.checkpoint()
+        time.sleep(2)
+
+        #
+        # Part 4: Ensure that all data is in both leader and follower.
+        #
+        cursor = session_follow.open_cursor(self.uri, None, None)
+        item_count = 0
+        while cursor.next() == 0:
+            item_count += 1
+        self.assertEqual(item_count, self.nitems * 6)
+        cursor.close()
+
+        cursor = self.session.open_cursor(self.uri, None, None)
+        item_count = 0
+        while cursor.next() == 0:
+            item_count += 1
+        self.assertEqual(item_count, self.nitems * 6)
+        cursor.close()
+
+        # FIXME: Remove this once the cleanup & unexpected log output are fixed.
+        self.ignoreStderrPatternIfExists('No such file or directory')
```

Comment:
This should is no longer required.

Reply:
thanks - i didn't have your changes in my branch. removed in 8caaba7